admin/option_values.php

```diff
@@ -26,12 +26,34 @@
 switch ($_GET['action']) {
   case ('update_sort_order'):
     foreach ($_POST['options_values_new_sort_order'] as $id => $new_sort_order) {
-
-      $db->Execute("UPDATE " . TABLE_PRODUCTS_OPTIONS_VALUES . "
-                    SET products_options_values_sort_order = " . (int)$_POST['options_values_new_sort_order'][$id] . "
-                    WHERE products_options_values_id = " . (int)$id);
+            if (empty($_POST['language']) || $_POST['language'] == 'current_only') {
+                $db->Execute(
+                    "UPDATE " . TABLE_PRODUCTS_OPTIONS_VALUES . "
+                    SET products_options_values_sort_order = " . (int)$new_sort_order . "
+                    WHERE products_options_values_id = " . (int)$id . " AND language_id = " . (int)$_SESSION['languages_id']
+                );
+            } else {
+                $languages_array = zen_get_languages();
+                foreach ($languages_array as $language) {
+                    $db->Execute(
+                        "UPDATE " . TABLE_PRODUCTS_OPTIONS_VALUES . "
+                    SET products_options_values_sort_order = " . (int)$new_sort_order . "
+                    WHERE products_options_values_id = " . (int)$id . " AND language_id = " . (int)$language['id']
+                    );
     }
-    $messageStack->add_session(SUCCESS_OPTION_VALUES_SORT_ORDER . ' ' . zen_options_name($_GET['options_id']), 'success');
+            }
+            $messageStack->add_session(
+                sprintf(
```

Comment:
Please don't change this.  The person doing this knows what he's changing and doesn't need all this feedback; the original simpler language define is enough.

Reply:
>The person doing this knows what he's changing and doesn't need all this feedback

I would not assume to know the profile of the average admin user. 
Sure people know what they want, or what they think they want, and do what they think they want to do, and what happens is what they expected, mostly.
I don't think you can have too much feedback. Not everyone is working in the admin all the time, and is 100% confident of what they are doing/have done. I certainly think twice and check locally before doing a global change.
Since it is already done, is it a negative thing to use it?

Reply:
Yes it is because it forces more language file editing on people who have already upgraded.  Please remove.

Reply:
> I'd like a second review of the improvements in this offering, given that many changes in the alpha seem to make concern about upgrading redundant.

Agreed on the point about the alpha.